ecs/backend.go

```diff
@@ -60,11 +76,34 @@ func service(ctx context.Context) (backend.Service, error) {
 }
 
 func getEcsAPIService(ecsCtx store.EcsContext) (*ecsAPIService, error) {
+	var region string
+	var profile string
+
+	if ecsCtx.CredentialsFromEnv {
+		creds := getEnvVars()
+		if !creds.haveRequiredCredentials() {
```

Comment:
actually, if this is the only place this is called from, we could do this check in getEnvVars and return an error, or rename haveRequiredCredentials to make it specific to env vars